CMakeLists.txt

```diff
@@ -1229,6 +1229,11 @@ SET(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
 SET(PACKAGE_VERSION "${LIB_VERSION}")
 IF(LIBTYPE STREQUAL "STATIC")
     SET(PKG_CONFIG_CFLAGS -DAL_LIBTYPE_STATIC ${PKG_CONFIG_CFLAGS})
+    FOREACH(LIB ${EXTRA_LIBS})
+        IF(NOT LIB MATCHES "^-.*")
+            SET(PKG_CONFIG_LIBS_PRIVATE "${PKG_CONFIG_LIBS_PRIVATE} -l${LIB}")
```

Comment:
As far as I can see, this will add ```-lwhatever``` for every entry in ```EXTRA_LIBS``` that doesn't start with ```-```. Is that correct? If so, I'm not sure it'll work correctly for entries that have a full path, such as the frameworks on OSX, or the result of the ```Find*``` scripts.

Reply:
> As far as I can see, this will add -lwhatever for every entry in EXTRA_LIBS that doesn't start with -. Is that correct?
Yes. Probably it's better to left out the if check?

Reply:
> Also does libraries return by Find* scripts include -l flags?

No, they give the full path and filename for the library (the base ```.so``` symlink, or the ```.a```, ```.dylib```, ```.framework```, ```.dll.a```, or ```.lib```, depending on the system and compiler). I honestly don't know what the correct way to handle it would be.